src/FSharp.Core/array.fs

```diff
@@ -2098,6 +2098,98 @@ module Array =
                 yield new ArraySegment<'T>(array, offset, maxIdxExclusive - offset)
             |]
 
+        let private createPartitionsUpTo maxIdxExclusive (array: 'T[]) =
+            createPartitionsUpToWithMinChunkSize maxIdxExclusive minChunkSize array
+
+        (* This function is there also as a support vehicle for other aggregations. 
+           It is a public in order to be called from inlined functions, the benefit of inlining call into it is significant *)
```

Comment:
How significant is significant?

Reply:
Was over 3x for the minBy operations IIRC.
(NotAtAllInlinedMinBy  vs InlinedUpToReduceByMinBy )
https://raw.githubusercontent.com/T-Gro/Array.Parallel.Aggs.Benchmark/main/Results/benchmark_results.md 

Inlining the lambda helps a lot in case of simple expressions.